src/test/kotlin/org/elm/ide/docs/ElmResolveLinkTest.kt

```diff
@@ -0,0 +1,37 @@
+package org.elm.ide.docs
+
+import com.intellij.psi.PsiManager
+import org.elm.lang.ElmTestBase
+import org.elm.lang.core.psi.ElmNamedElement
+import org.intellij.lang.annotations.Language
+
+class ElmResolveLinkTest : ElmTestBase(){
+    fun `test type`() = doTest(
+"""
+type Foo = Bar
+   --X
+
+foo : Foo
+foo = 0
+--^
+""", "Foo")
+
+    fun `test type alias`() = doTest(
+            """
+type alias Foo = Int
+         --X
+
+foo : Foo
+foo = 0
+--^
+""", "Foo")
```

Comment:
I wanted to add a test like this:

```kotlin
    fun `test qualified type`() = doTest(
            """
--@ Foo.elm
module Foo exposing (Bar)

type Bar = Bar
   --X

--@ main.elm
import Foo

foo : Foo.Bar
foo = 0
--^
""", "Foo.Bar")
```

But I couldn't get the `fileTreeFromText` process working with a marker in two different files. Let me know if you have an idea on how to set this up.